METRICS.md

```diff
@@ -0,0 +1,282 @@
+# Four Key Metrics Calculations
+
+![Image of the Four Keys dashboard.](images/dashboard.png)
```

Comment:
Also add this image in the readme file under "dashboard" (and if you update to include more data points, just keep the images the same in these two files).